docs/README.md

```diff
@@ -204,8 +204,14 @@ binaries you intend to use.
 },
 ```
 
+You can set the `ANALYZERS_FROM_PATH` environment variable before running a
+CodeChecker command to `YES` or `1` to enforce taking the analyzers from the
```

Comment:
In this documentation `YES` is expected, but in `get_analyzer_from_path()` you find `yes`. Check `package_layout.md` too.